### PR TITLE
Added workaround for dockerd's max layer depth

### PR DESCRIPTION
**Pull request by ncarlson**

### CODE REVIEW

File: tanzu-build-service/troubleshooting.hbs.md [hunk 1]
```diff
@@ -68,3 +68,30 @@ Tanzu Application Platform uses the default storage class which uses EBS volumes
...
+     "harbor.somewhere.com/aws-repo/build-service:clusterbuilder-full@sha256:065bb361fd914a3970ad3dd93c603241e69cca214707feaa6
+     d8617019e20b65e":  rpc error: code = Unknown desc = failed to register layer: max depth exceeded
+   ```
+
+If you see the words, "max depth exceeded", you have likely encountered dockerd's maximum image layer depth limitation.
+
+### Solution
+
+To work around this issue, configure your cluster to use containerd as its default container runtime. Please refer to 
+your Kubernetes cluster provider's documentation for specific instructions.
```
Comment (pviraj59) [inline tanzu-build-service/troubleshooting.hbs.md:97]: should we add link to the default documentation  for most used providers?

**Comment by RichardJJG** [comment]
@ncarlson, please comment here, tagging @snneji, when this is ready to merge.

File: tanzu-build-service/troubleshooting.hbs.md (hunk 1 above)
Comment (ncarlson) [reply tanzu-build-service/troubleshooting.hbs.md:97]: Good idea! Added.

**Review by tomkennedy513 — APPROVED** [review]

**Review by tylerphelan — APPROVED** [review]

**Comment by ncarlson** [comment]
@snneji, this PR is ready to merge.

**Comment by snneji** [comment]
Hi @ncarlson just double checking, is this only for 1.3?

**Pull request merged**